### DISCUSSION TITLE
主题设置更改后点击更改设置按钮没有任何反应

### DISCUSSION THREAD

**Discussion by Onsaki** [solstice23/argon-theme#465] [Q&A]
主题设置里面改一些设置之后点击保存设置没有反应了，不知道是不是我个例。
wordpress版本是5.9.3
出现问题前的异常操作就只有整站迁移，是通过在插件商店搜索“迁移”后结果的第一个插件整站迁移过来的，最近看到的唯一异常就是进到wp后台顶部会有个让我点击什么邮件里面的链接证明我是人的提示，但是我翻我邮箱又没翻到那个，所以就把那个提示条关了。

**Comment by vite666** [comment]
刷新 redis 或其它什么的缓存
